src/test/scala/org/broadinstitute/dsde/firecloud/service/DataUseRestrictionTestFixtures.scala

```diff
@@ -37,7 +36,7 @@ object DataUseRestrictionTestFixtures {
     mkWorkspace(attributes, code, s"{${code.replace("-","")}}-unique")
   }
 
-  val listCodes: Seq[String] = Seq("RS-POP")
+  val listCodes: Seq[String] = Seq()
```

Comment:
because we removed RS-POP. we don't have any generic lists to test. (we do test disease lists with a different, more specific fixture). if we re-introduce a code that contains a list, we will probably want these tests again. but currently we don't need them.